File: Javascript/MaiorMenor.js
const readline = require("readline");

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});

console.log("_________ Digite 10 numeros _________");
const numeros = [];

function perguntarNumero(i) {
    if (i >= 10) {
        console.log("Numeros digitados: ");
        console.log(numeros);
        rl.close();
        return;
    }

    rl.question("Digite numero: ", function(numero) {
        numeros.push(Number(numero));
        perguntarNumero(i + 1);
    });
}

perguntarNumero(0);

File: Python/Logic-execicis/MaiorMenor.py
print(" ---------------- Digite 10 numeros ---------------- ")

numeros = []
numeroMaior = 0
numeroMenor = 0

for numero in range(10):
    numeros.append(int(input("Digite um numero: ")))

    numeroMaior = numeros[0]
    numeroMenor = numeros[0]

for numero in numeros:
    if numero > numeroMaior:
        numeroMaior = numero
    if numero < numeroMenor:
        numeroMenor = numero

print("Maior numero é: ", numeroMaior)
print("Menor numeor é: ", numeroMenor)